packages/toolkit/src/query/core/buildSlice.ts

```diff
@@ -505,10 +526,12 @@ export function buildSlice({
     ...configSlice.actions,
     ...querySlice.actions,
     ...subscriptionSlice.actions,
+    ...internalSubscriptionsSlice.actions,
     ...mutationSlice.actions,
     /** @deprecated has been renamed to `removeMutationResult` */
     unsubscribeMutationResult: mutationSlice.actions.removeMutationResult,
     resetApiState,
+    actuallyMutateSubscriptions,
```

Comment:
I'm not sure if we should have that on `actions` here

Reply:
The other option would be to just move it straight over into the file where it's used. This was at least partly laziness :)

Reply:
^^